object-search/flow-index.yml

```diff
@@ -0,0 +1,31 @@
+!Flow
+with:
+  logserver: false
+pods:
+  crafter:
+    uses: pods/craft.yml
+    read_only: true
+  encoder:
+    uses: pods/encode.yml
+    parallel: $PARALLEL
+    timeout_ready: 600000
+    read_only: true
+  vec_idx:
```

Comment:
I would expect here to have a compound_indexer for `chunk_vec` and `chunk_kv` data, followed by another pod with `doc_kv` index. In this case you will have to be careful how to connect the pods since it is important to know what needs and does not need preprocessing.